src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -491,10 +542,11 @@ def helm_install_release(chart_path, subscription_id, kubernetes_distro, resourc
     response_helm_install = Popen(cmd_helm_install, stdout=PIPE, stderr=PIPE)
     _, error_helm_install = response_helm_install.communicate()
     if response_helm_install.returncode != 0:
-        if 'forbidden' in error_helm_install.decode("ascii"):
+        if ('forbidden' in error_helm_install.decode("ascii") or 'timed out waiting for the condition' in error_helm_install.decode("ascii")):
             telemetry.set_user_fault()
         telemetry.set_exception(exception=error_helm_install.decode("ascii"), fault_type=consts.Install_HelmRelease_Fault_Type,
                                 summary='Unable to install helm release')
+        logger.warning("Please check if the azure-arc namespace was deployed and run 'kubectl get pods -n azure-arc' to check if all the pods are in running state.")
         raise CLIError("Unable to install helm release: " + error_helm_install.decode("ascii"))
```

Comment:
Should we add one of the possible causes could be resource scheduling? or this is enough?

Reply:
Updated the message with a possible cause